src/test/resources/movie-tests.adoc

```diff
@@ -996,21 +1098,22 @@ mutation {
 [source,json]
 ----
 {
-  "fromName": "Action",
-  "toMovies": [
-    "m1"
+  "createUserUserId": 1,
+  "createUserValidTypes": [
+    "User"
   ]
 }
 ----
 
 .Cypher
 [source,cypher]
 ----
-MATCH (from:Genre { name: $fromName })
-MATCH (to:Movie) WHERE to.movieId IN $toMovies
-MERGE (from)<-[:IN_GENRE]-(to)
-WITH DISTINCT from AS addGenreMovies
-RETURN addGenreMovies { .name } AS addGenreMovies
+CREATE (createUser:User:Person { userId: $createUserUserId })
+WITH createUser
+RETURN createUser {
+  .userId,
+  __typename: head( [ label IN labels(createUser) WHERE label IN $createUserValidTypes ] )
```

Comment:
what was this again?

Reply:
This is to have the name of the type when using interfaces, so that you know what concrete type the current entry is of.